cmd/admin-handlers_test.go

```diff
@@ -47,6 +47,16 @@ func prepareAdminXLTestBed() (*adminXLTestBed, error) {
 	// reset global variables to start afresh.
 	resetTestGlobals()
 
+	// Set globalIsXL to indicate that the setup uses an erasure
+	// code backend.
+	globalIsXL = true
+
+	// Init global heal state
+	if globalIsXL {
```

Comment:
What is the point of checking it, if it is always true?

Reply:
> What is the point of checking it, if it is always true?

oups, fixed!